src/gui/accountmanager.cpp

```diff
@@ -321,6 +323,10 @@ void AccountManager::saveAccountHelper(Account *acc, QSettings &settings, bool s
     settings.setValue(QLatin1String(serverVersionC), acc->_serverVersion);
     settings.setValue(QLatin1String(serverColorC), acc->_serverColor);
     settings.setValue(QLatin1String(serverTextColorC), acc->_serverTextColor);
+
+    // TODO: serverHasValidSubscription is per user or general?
+    settings.setValue(QLatin1String(serverHasValidSubscriptionC), acc->capabilities().serverHasValidSubscription());
+
```

Comment:
Why are we storing this as a setting? Shouldn't we just fetch this from capabilities on startup?